project/Dependencies.scala

```diff
@@ -32,6 +32,7 @@ object Dependencies {
     "io.spray"                      %% "spray-client"              % sprayV,
     "io.spray"                      %% "spray-routing-shapeless23" % sprayV,
     "io.spray"                      %% "spray-testkit"             % sprayV   % "test",
+    "net.virtual-void"              %% "json-lenses"               % "0.6.2"  % "test",
```

Comment:
This is used in `LibraryApiServiceSpec` to look into the JSON response. Pretty nice!

Reply:
this is great info! we might want to use that instead of the spray json stuff when we implement the entity service